failure in ud/test_ud.ack_req_single/1 with valgrind

http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/4814/label=hpc-test-node2,worker=0/console
```
00:38:15 [ RUN      ] ud/test_ud.ack_req_single/1
00:38:16 /scrap/jenkins/scrap/workspace/hpc-ucx-pr-2/label/hpc-test-node2/worker/0/contrib/../test/gtest/uct/ib/test_ud.cc:330: Failure
00:38:16 Value of: ack_req_tx_cnt
00:38:16   Actual: 4
00:38:16 Expected: 2
00:38:16 /scrap/jenkins/scrap/workspace/hpc-ucx-pr-2/label/hpc-test-node2/worker/0/contrib/../test/gtest/uct/ib/test_ud.cc:331: Failure
00:38:16 Value of: tx_ack_psn
00:38:16   Actual: 2
00:38:16 Expected: 1
00:38:16 
[  FAILED  ] ud/test_ud.ack_req_single/1, where GetParam() = ud/mlx5_0:1 (372 ms)
...
00:42:10 [  FAILED  ] 1 test, listed below:
00:42:10 [  FAILED  ] ud/test_ud.ack_req_single/1, where GetParam() = ud/mlx5_0:1
00:42:10 
00:42:10  1 FAILED TEST
00:42:14 
make: *** [test_valgrind] Error 1
```

--- a/test/gtest/uct/ib/test_ud.cc
+++ b/test/gtest/uct/ib/test_ud.cc
@@ -314,6 +314,8 @@ UCS_TEST_P(test_ud, tx_window2) {
 UCS_TEST_P(test_ud, ack_req_single) {
 
     connect();
+    disable_async(m_e1);
+    disable_async(m_e2);
     set_tx_win(m_e1, 2);
     ack_req_tx_cnt = 0;
     tx_ack_psn = 0;

